include/fastdds/dds/core/policy/ParameterTypes.hpp

```diff
@@ -976,6 +976,22 @@ class ParameterProperty_t
 
 };
 
+/**
+ * Parameter property ID for persistence GUID
+ * @ingroup PARAMETER_MODULE
+ */
+const std::string parameter_property_persistence_guid = "PID_PERSISTENCE_GUID";
+/**
+ * Parameter property ID for participant type
+ * @ingroup PARAMETER_MODULE
+ */
+const std::string parameter_property_participant_type = "PARTICIPANT_TYPE";
+/**
+ * Parameter property ID for Discovery Server version
+ * @ingroup PARAMETER_MODULE
+ */
+const std::string parameter_property_ds_version = "DS_VERSION";
+
```

Comment:
Should we have an additional const for the current DS version?

Reply:
Most definitely, good idea!